src/workbench/extensions/manager/components/manager/packCard/PackCard.test.ts

```
@@ -1,6 +1,7 @@
+import { createTestingPinia } from '@pinia/testing'
 import { mount } from '@vue/test-utils'
-import { createPinia, setActivePinia } from 'pinia'
 import Card from 'primevue/card'
```

ESLint

[error] 'Card' is defined but never used.

(unused-imports/no-unused-imports)